freqtrade/persistence/models.py

```diff
@@ -333,7 +333,7 @@ def __init__(self, **kwargs):
         for key in kwargs:
             setattr(self, key, kwargs[key])
         if self.isolated_liq:
-            self.set_isolated_liq(self.isolated_liq)
+            self.set_isolated_liq(isolated_liq=self.isolated_liq)
```

Comment:
I'd not use the "isolated_liq" parameter here.
the trade-object should have one liquidation price (if it's cross or isolated should not matter, and won't have to be stored in a separate variable).